app/_how-tos/manage-secrets-in-insomnia-with-1password.md

```diff
@@ -0,0 +1,109 @@
+  - q: Why am I getting errors when using the 1Password plugin?
+    a: |
+        There are a few known issues that may be cause errors with the plugin. Make sure that everything is configured properly:
+        * [Integrate 1Password CLI with the 1Password desktop app](https://developer.1password.com/docs/cli/get-started/#step-2-turn-on-the-1password-desktop-app-integration).
+        * On macOS, set the `cliPath` parameter to the correct path to the 1Password CLI binary, and make sure that the desktop app is set to [run in the background](https://developer.1password.com/docs/cli/app-integration/#if-you-see-a-connection-error).
+
+---
+
+## Enable elevated access for plugins
+
+The 1Password plugin needs to make system-level calls to the local 1Password CLI. To allow it to do this, go to **Preferences** > **Plugins** and select the **Allow elevated access for plugins** checkbox.
+
+## Install the 1Password plugin
+
+Go to the [Insomnia Plugin Hub](https://insomnia.rest/plugins/insomnia-plugin-op) and install the 1Password plugin.
+
+## Configure the plugin
+
+The 1Password plugin is configured through a JSON [environment variable](/insomnia/environments/). This environment variable can be defined with any environment type. In this example, we'll configure it in a global base environment.
+
+1. In your project, create a new environment.
+1. In the base environment, add the following content, with the correct path and account name:
+   ```json
+   {
+    "__op_plugin": {
+      "cliPath": "/opt/homebrew/bin/op",
+      "defaultAccount": "account-name.1password.com"
+    }
+   }
+   ```
+
+   {:.info}
+   > **Notes**:
+   > * The `cliPath` parameter is the path to the 1Password CLI binary. This parameter is only required on macOS to avoid issues locating 1Password CLI.
+   > * The plugin also supports the `cacheTTL` and `flags` parameters. See the [plugin configuration](https://insomnia.rest/plugins/insomnia-plugin-op#configuration) for more details.
+
+## Use a 1Password secret in a request
+
+The 1Password plugin creates a custom [template tag](/insomnia/template-tags/) that you can configure to fetch a secret and use it in multiple places in a request.
+
+1. Create a collection or open an existing one.
+1. Click the environments on the left pane and select the global environment containing the 1Password configuration that we created in the previous step.
+1. Create a new request. For this example, we'll send a `POST` request to `http://httpbin.konghq.com/anything`.
+1. Open the **Body** tab, click **No Body** and select **Form Data**.
+1. Enter a parameter name, `secret` for example.
+1. Click the **value** field field, press `Control+Space` to display the available template tags, and select **1Password => Fetch Secret**.
+1. Click the template tag to configure it, and add the reference to the 1Password secret we created in the [prerequisites](#1password): `op://insomnia/test-secret/password`. The live preview should show `my-password`.
```

Comment:
I think something went wrong in this step that gave me an error? I used the settings gear icon and selected the values from that, should I have pasted the values I put in my environment variables instead? I also used our Kong account, maybe I don't have the right permissions?

Reply:
I'm getting `parseSignature: expected comma after expression`

Reply:
I'm not sure what you mean, there was no need to use th gear icon, just paste the reference in the reference field, can you try again?